src/features/MovieList/index.js

```diff
@@ -22,15 +24,22 @@ const MovieList = () => {
   }, [dispatch, status]);
 
   return (
-    <MovieListOrganizer>
-      {moviesList.map((movie) => (
-        <MovieTile
-          key={movie.id}
-          movie={movie}
-          movieGenres={movieGenres}
-        />
-      ))}
-    </MovieListOrganizer>
+    status === "success" ? (
```

Comment:
The first time you click on MovieList or People, etc., the Loading displays perfectly. However, when you want to enter People or another component with this condition again, the loading window doesn't show up. Maybe it's worth adding a condition right with this pull request that always triggers Loading until the entire content is loaded?